tests/unit/commands/test_remote.py

```diff
@@ -435,3 +439,121 @@ def test_run_in_repo_newer_than_core22(emitter, mock_run_legacy, monkeypatch, ne
     emitter.assert_debug(
         "Running fallback remote-build because new remote-build is not available."
     )
+
+
+##################
+# Build id tests #
+##################
+
+# The build-id is not currently used, so these unit tests test the log output.
+# When #4323 is complete, these tests can be rewritten to verify the build-id passed
+# to the new remote-build code.
+
+
+@pytest.mark.parametrize(
+    "create_snapcraft_yaml", CURRENT_BASES - {"core22"}, indirect=True
```

Comment:
remind me please: why doesn't this use core22?

Reply:
No reason, I need to test this on all bases.